PostHog/PostHogFeatureFlags.swift

```diff
@@ -34,7 +34,7 @@ class PostHogFeatureFlags {
         distinctId: String,
         anonymousId: String,
         groups: [String: String],
-        completion: @escaping ([String: Any]?, [String: Any]?) -> Void
+        callback: @escaping () -> Void
```

Comment:
Since we have special logic to check if the feature flag is enabled and so on, it's better to not expose the raw dictionary, otherwise it's too easy to misuse the API, I missed that before.

Reply:
Agreed